h1b/src/nvcounter/capsule.rs

```diff
@@ -0,0 +1,293 @@
+// Copyright 2019 Google LLC
```

Comment:
Generally, capsule refers to a property of the code (a capsule has no unsafe code). But all of this code seems to be capsule code (e.g., the internal code too). I don't really have a better name, except maybe ::driver?